index.html

```diff
@@ -352,7 +366,7 @@ <h3>
 							<p class="milestone"><b>Expected completion:</b> Q1 2023</p>
 						</dd>
 
-						<dt id="a11y" class="spec">EPUB Accessibility</dt>
+						<dt id="a11y" class="spec">EPUB Accessibility 1.X</dt>
 						<dd>
               				<p>
 								The next version of the EPUB Accessibility defines discovery and content accessibility requirements for EPUB Publications. The specifications is build on <a href='https://www.w3.org/TR/WCAG22/'>WCAG 2.x</a> and adds accessibility requirements specific to publishing.
```

Comment:
Typo: "The specifications is builds on"
Fix with either "The specification is built on" or "The specification builds on"

Reply:
Done in https://github.com/w3c/epub-3-wg-charter/pull/47/commits/8474c3fe21cb73e77329c0b35f368553ac523178, thanks!